packages/forklift-console-plugin/src/modules/Providers/utils/components/DetailsPage/DetailItem.tsx

```diff
@@ -177,8 +179,9 @@ export const DescriptionTitle: React.FC<{ title: string }> = ({ title }) => (
 export const ContentField: React.FC<{
   content: ReactNode;
   onEdit: () => void;
-}> = ({ content, onEdit }) =>
-  onEdit ? (
+  canEdit?: boolean;
+}> = ({ content, onEdit, canEdit = !!onEdit }) =>
```

Comment:
In this way, u are keeping the same logic?
Could you reinforce the condition of canEdit && onEdit? And giving canEdit = true as default value,
In this way, you are not breaking anything else, just expanding the component.

Reply:
That would work the same but I can change it